js-exercises/.directory

```diff
@@ -0,0 +1,6 @@
+[Dolphin]
+Timestamp=2020,3,15,12,44,55
```

Comment:
I am not aware of what these files are doing personally, but could have been taken out from the code branch and pushed separately from other branches.

Reply:
ok cool man thanks for the quick reply. I will make sure to do that next time.

thanks once again